code-challenges.js

```diff
@@ -30,3 +119,31 @@ var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 var testingArray3 = [3, 7, "hi", 10, 3, "hello", 4, "hi"]
 var testingArray4 = [7, "hi", 3, 1, "hi", 4, "hello", 4, 7]
 // Expected output: [ 3, 7, "hi", 10, "hello", 4, 1 ]
+
+//when I tried it out the first time, I kept getting an array with only one value = 1. The only unique value in the group 
+
+// arrayCombiner = (arr1, arr2) => {
+//     //this combines all the values into arr1
+//   arr1.push(...arr2)
+//   //sets an empty array to push the non
+//   let arr3 = []
+//   arr1.map((value, index) => {
+//     if (arr1.indexOf(value) === index){
+//       arr3.push(value)
+//     }
+//   })
+// }
+// //   return arr3.push(
+// //     arr1.filter((value, i) => arr1.indexOf(value) === i)
+// //   )
+// // }
+// console.log(arrayCombiner(testingArray3, testingArray4))
+
+arrayCombiner2 = (arr1, arr2) => {
+  //this combines all the values into arr1
+arr1.push(...arr2)
+//filters and returns the new array. takes in the value and index and then filters out the values if they occur in the array more than once. it will check if the value occurs at more than one index
+  return arr1.filter((value, i) => arr1.indexOf(value) === i)
+}
+
+console.log(arrayCombiner2(testingArray3, testingArray4))
```

Comment:
this how I did mine
const noDuplicates = (arr1, arr2) => {
	let uniqueArray = arr1.concat(arr2);
	return uniqueArray.filter(
		(item, index) => uniqueArray.indexOf(item) === index
	);
};

console.log(noDuplicates(testingArray3, testingArray4));